rpmlint/checks/PAMModulesCheck.py

```diff
@@ -8,7 +8,10 @@ class PAMModulesCheck(AbstractCheck):
 
     def __init__(self, config, output):
         super().__init__(config, output)
-        self.pam_whitelist = config.configuration['PAMModulesWhiteList']
+        self.pam_authorized_modules = config.configuration['PAMAuthorizedModules']
+        if not self.pam_authorized_modules:
+            # compatibility: read the values from the original configuration option
```

Comment:
This is a functional change and should go in a separate pull request.

Reply:
ok, I'll split the commits into 2 pull requests.

Reply:
May I please remind this.